### PR TITLE
Add pt-br/docs/contribute/review/reviewing-prs.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the /docs/contribute/review/reviewing-prs.md

### CODE REVIEW

File: content/pt-br/docs/contribute/review/reviewing-prs.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    Se um autor renomear uma página ou combinar duas páginas, nós (_Kubernetes SIG Docs_) geralmente evitamos pedir a esse autor que corrija todas as questões gramaticais ou ortográficas que poderíamos identificar dentro desse conteúdo movido.
+- Existem palavras complicadas ou arcaicas que podem ser substituídas por uma palavra mais simples?
+- Existem palavras, termos ou frases em uso que podem ser substituídos por uma alternativa não discriminatória?
+- A escolha da palavra e sua capitalização seguem o [guia de estilo](/docs/contribute/style/style-guide/)?
+- Existem frases longas que podem ser mais curtas ou menos complexas?
+- Existem parágrafos longos que podem funcionar melhor como uma lista ou tabela?
+
+### Conteúdo
+
+- Existe o conteúdo semelhante em outro lugar no site Kubernetes?
```
Comment (stormqueen1990) [inline content/pt-br/docs/contribute/review/reviewing-prs.md:132]: Nit: "conteúdo" doesn't need an article preceding it in this sentence.

**Review by stormqueen1990 — COMMENTED** [review]
Hi there, @MrErlison! 🎉 

Thank you very much for your contribution!

I found some things that need to be addressed, please feel free to reach out should you have any questions.

File: content/pt-br/docs/contribute/review/reviewing-prs.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+## Antes de começar
+
+Antes de começar uma revisão:
+
+- Leia o [Código de Conduta da CNCF](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) e certifique-se de cumpri-lo o tempo todo.
+- Seja educado, atencioso e prestativo.
+- Comente os aspectos positivos dos PRs, bem como mudanças.
+- Seja empático e cuidadoso, observe como sua avaliação pode ser recebida.
+- Assuma boas intenções e faça perguntas esclarecedoras.
+- Colaboradores experientes, considere emparelhar com os novos colaboradores cujo trabalho requer grandes mudanças.
```
[the remark this answers is not in the document]
Comment (MrErlison) [reply content/pt-br/docs/contribute/review/reviewing-prs.md:32]: Really, I think "trabalhar em par", but write "emparelhar". Thank you for your eagle eye.

**Comment by edsoncelio** [comment]
/lgtm
cc @stormqueen1990

**Comment by devlware** [comment]
Thank you @MrErlison. 👏🏻

**Pull request merged**